schemas/pom.xml

```diff
@@ -8,9 +8,9 @@
     <packaging>jar</packaging>
```

Comment:
Should this change be in this PR?

Reply:
This change is in another PR though right? Can that one go in first?

Reply:
I think this comment stream is for a change that no longer exists in the branch, @MattCline-SL  can you re-review the branch from its current state. There are no `pom.xml` changes in this PR

Reply:
Yep this is a GitHub problem - I couldn't tell that from the reply to my comment. I'm still not that used to GitHub's pull request interface.